config/build.ts

```diff
@@ -76,13 +61,14 @@ const defaultApiSets = {
 })();
 
 
-async function processSnippets(dir, processedSnippets) {
+async function processSnippets(processedSnippets) {
     return new Promise((resolve, reject) => {
         banner('Loading & processing snippets');
-        let files$ = getFiles(path.resolve(dir), path.resolve(dir));
+        let files$ = getFiles(path.resolve('private-samples'), path.resolve('private-samples'));
```

Comment:
I would put `private-samples` and `samples` as constants somewhere at the top of the file